package.json

```diff
@@ -24,6 +24,7 @@
     "@justeat/f-header": "1.1.0",
     "@justeat/f-icons": "1.16.0",
     "@justeat/f-toggle": "1.1.0",
+    "@justeat/f-utilities": "0.5.0",
```

Comment:
v0.6.0 looks to be the latest version with cookie message

Reply:
Actually I am going to update the component with IE bug fix so can for now be as it is